apps/prairielearn/src/tests/e2e/reportCheating.spec.sql

```
@@ -0,0 +1,109 @@
+  pt_reservations (
+    access_start,
+    access_end,
+    enrollment_id,
+    exam_id,
+    session_id
+  )
+SELECT
+  NOW() - INTERVAL '1 minute',
+  NOW() + INTERVAL '1 hour',
+  enrollment.id,
+  exam.id,
```

SQLFluff (4.2.2)

[error] Reuse of column alias 'id' from line 49.

(AL08)